incubator/elasticsearch/values.yaml

```diff
@@ -4,12 +4,13 @@
 appVersion: "5.4"
 
 image:
-  repository: "centerforopenscience/elasticsearch"
-  tag: "5.4"
+  repository: "blacktop/elasticsearch"
```

Comment:
Any particular reason why you wouldn't use the official upstream images? I'd say a generic helm chart should prefer the official images, probably.

Reply:
I was wondering the same thing.

Reply:
Main reason is that the official images install a trial version of X-Pack and are about 3x the size of this image. I was however going back and forth between the official image and this one. This pick was made as a personal recommendation rather than "I want to make sure I'm on good terms with upstream companies". I'm willing to leave the pick of default image up to a democratic process. :)

Reply:
Sounds like a good reason to me.

Reply:
I'd prefer to use official docker images: https://www.docker.elastic.co/